profiles/logging.py

```diff
@@ -0,0 +1,38 @@
+import logging
+import os
+import sys
+from typing import List
+
+from pythonjsonlogger import jsonlogger
+
+
+class CustomJsonFormatter(jsonlogger.JsonFormatter):
+    def formatException(self, ei: List) -> dict:
+        info_type, info_value, info_traceback = sys.exc_info()
```

Comment:
Is this meant to not use `ei`?

Reply:
no, it should be unpack that rather than call again `sys.exc_info()`